CMakeLists.txt

```diff
@@ -353,16 +382,22 @@ if(BUILD_ONNX_PYTHON)
                              ${PYTHON_INCLUDE_DIR})
 
   # pybind11 is a header only lib
-  find_package(pybind11 2.2)
-  if(pybind11_FOUND)
-    target_include_directories(onnx_cpp2py_export PUBLIC
-      ${pybind11_INCLUDE_DIRS})
+  if (HUNTER_ENABLED)
+    hunter_add_package(pybind11)
+    find_package(pybind11 CONFIG REQUIRED)
+    target_link_libraries(onnx_cpp2py_export PUBLIC pybind11::pybind11)
   else()
-    if(EXISTS ${ONNX_ROOT}/third_party/pybind11/include/pybind11/pybind11.h)
+    find_package(pybind11 2.2)
+    if(pybind11_FOUND)
       target_include_directories(onnx_cpp2py_export PUBLIC
-        ${ONNX_ROOT}/third_party/pybind11/include)
+        ${pybind11_INCLUDE_DIRS})
```

Comment:
We are not entering the `else` block, you can keep the code to minimize number of changes in patch.